### PR TITLE
chore(web): added full list of vars to env production file

### PR DESCRIPTION
**Pull request by francescobrenci-AR**
## Describe your changes
Added completed list of Vars needed in the env.production file

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/.env.production [hunk 1]
```diff
@@ -70,3 +70,11 @@ LOG_LEVEL_FILE=debug
 # The logging level for the stdout.
 # Options: trace, debug, info, warn, error, fatal, silent
 LOG_LEVEL_STDOUT=info
+
+## apps/api ###################################################################
+
+# Required values as api appeal formatters imported by the web tests are are
+# coupled to config files (for some reason).
+# TODO: remove the requirement for these props in the web .env config
+DATABASE_URL="sqlserver://"
```
Comment (mayya-bond-kc) [inline apps/web/.env.production:79]: You shouldn't need the DATABASE_URL

**Review by mayya-bond-kc — CHANGES_REQUESTED** [review]

File: apps/web/.env.production (hunk 1 above)
Comment (francescobrenci-AR) [reply apps/web/.env.production:79]: so this is a carry over from our env.test, @lutamKC added it there, I'm assuming we might nee them here as well?
@lutamKC can you confirm

**Comment by lutamKC** [comment]
i'm not sure what's happening here actually. where the lowercase values come from?

**Comment by francescobrenci-AR** [comment]
> i'm not sure what's happening here actually. where the lowercase values come from?

which lowercase values?

File: apps/web/.env.production (hunk 1 above)
Comment (francescobrenci-AR) [reply apps/web/.env.production]: removed the vars for now

**Review by mayya-bond-kc — APPROVED** [review]

**Pull request merged**